From da9eba89c3ba63c9696c3c559a06d17ddaf5935e Mon Sep 17 00:00:00 2001
From: "Srivatsa S. Bhat (VMware)" <srivatsa@csail.mit.edu>
Date: Fri, 31 Mar 2023 14:19:10 -0700
Subject: [PATCH 2/2] linux-secure, Makefile: Add kernel flavor info to the
 generated version.h file

Add "PHOTON_KERNEL_FLAVOR_SECURE" macro definition to the generated file
include/generated/uapi/linux/version.h to identify Photon's
linux-secure kernel flavor.

Signed-off-by: Srivatsa S. Bhat (VMware) <srivatsa@csail.mit.edu>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 55f1c4f1c8c1..13788f35caad 100644
--- a/Makefile
+++ b/Makefile
@@ -1349,7 +1349,7 @@ define filechk_version.h
 	echo '#define PHOTON_KERNEL';                                                   \
 	echo '#define PHOTON_KERNEL_RELEASE                                             \
 	$(shell echo -n "$(KERNELRELEASE)" | cut -d'-' -f2 | cut -d'.' -f1)';           \
-	echo '#define PHOTON_KERNEL_FLAVOR_'
+	echo '#define PHOTON_KERNEL_FLAVOR_SECURE'
 endef
 
 $(version_h): PATCHLEVEL := $(or $(PATCHLEVEL), 0)
-- 
2.25.1

